qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -3,9 +3,6 @@
     only Windows
     virtio_win_media_type = iso
     cdroms += " virtio"
-    clone_master = yes
-    master_images_clone = image1
-    remove_image_image1 = yes
```

Comment:
Why do this? Does it relate to enable virtiofs debug info?
If not, we could add another commit for it in this PR.

Reply:
Soorry for my careless, it was a wrong push.
Updated.@YonxueHong

Reply:
> Soorry for my careless, it was a wrong push. Updated.@YonxueHong

Never mind, I see it.